images/bootstrap/resources/proxy.yaml

```diff
@@ -49,3 +55,11 @@ spec:
             port: 5432
           initialDelaySeconds: 10
           timeoutSeconds: 5
+        volumeMounts:
+          - mountPath: /etc/etcd/secrets
+            name: etcd-secrets
+      volumes:
+        - name: etcd-secrets
+          hostPath:
+            path: /var/state
```

Comment:
No, it's in planet, so we should pass it to container somehow.

You can test it like
```
$ cat /tmp/nginx.yaml 
apiVersion: v1
kind: Pod
metadata:
  name: nginx
spec:
  containers:
  - name: nginx
    image: nginx
    ports:
    - containerPort: 80
  hostNetwork: true
```